browser/ipfs/ipfs_tab_helper_delegate_impl.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_IPFS_IPFS_TAB_HELPER_DELEGATE_IMPL_H_
+#define BRAVE_BROWSER_IPFS_IPFS_TAB_HELPER_DELEGATE_IMPL_H_
+
+#include "brave/components/ipfs/browser/ipfs_tab_helper_delegate.h"
+
+namespace ipfs {
+
+class IpfsTabHelperDelegateImpl : public IpfsTabHelperDelegate {
```

Comment:
I think we can merge all delegates into one interface, as most components do. Given that all of them are quite tiny

Reply:
Removed the service one so resolving this.

Reply:
delegates are gone now in any case.